lib/config/core.js

```diff
@@ -76,6 +76,7 @@ function load () {
   cb = once(function (er, conf) {
     if (!er)
       exports.loaded = conf
+      loading = false
```

Comment:
This... _was_ supposed to be set back to `false` eventually... right? I need this so that I can force `npmconf` to re-load in the test.